docker-compose.yml

```
@@ -56,6 +56,13 @@ services:
             - 1025:1025
             - 8025:8025
 
+    mitmproxy:
+        image: mitmproxy/mitmproxy
```

```suggestion
        image: mitmproxy/mitmproxy:12
        pull_policy: weekly
```

Now Renovate can do major upgrades and users get updates weekly